specs/error_state/ErrorCodeStore.md

```diff
@@ -1,54 +1,40 @@
 # ErrorCodeStore state
 
 ## Procedure
-`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and 
-`MaxCodeSizeExceeded`.
-this type of error only occurs when executing create(create,create2) op code or tx deploy 
-transaction(tx.to = null).
+`ErrorCodeStore` is a combined error code and it's designed to handle code store related errors: `CodeStoreOutOfGas` and `MaxCodeSizeExceeded`. This type of error only occurs when executing `CREATE`/`CREATE2` opcode or a deployment transaction (tx.to = null).
 
 ### EVM behavior
-When handling a CREATE-kind transaction, Initial bytecode opcodes will run and current call context is created.
-The final bytecode opcodes to store for new contract is the `RETURN` opcode of init codes result.
+While handling a `CREATE`/`CREATE2` opcode, initial bytecode will be executed and then current call context is created. The contract bytecode will be returned through `RETURN` opcode as the execution result. `RETURN` returns a specified memory chunk [`offset`...`offset` + `length`] as bytecode. The gas cost for storing the bytecode is:
```

Comment:
I find this paragraph a bit hard to read. Here is a suggestion (of my understanding), please correct me if I am wrong.